hack/benchmark/README.md

```diff
@@ -6,9 +6,39 @@ Benchmark 3-member etcd cluster to get its read and write performance.
 
```

Comment:
(Nitpick) This file should also adopt the Headings Hierarchy described [here](https://github.com/joshix/docs/blob/styledir-hdings/CONTRIBUTING.md#headings). Minimally, it should begin with an appropriate level-one (`#`) heading.